README.md

```diff
@@ -96,7 +96,13 @@ On Fedora the dependencies are:
  * nspr-devel
  * pkgconfig
 
-On Osx the denpendencies are: (which can be installed by homebrew)
+On Debian, the dependencies can be installed with the following command:
+
+```bash
+apt-get install libssl-dev gnutls-dev libnss3-dev libpcap-dev libev-dev libnspr4-dev pkgconf
```

Comment:
The last one should be `pkg-config`. `pkgconf` seems to be a replacement, but I didn't hear of it until now.

Reply:
You got it!

Indeed it's a replacement that I prefer to work with, more or less a drop in for pkg-config, works with both nonetheless.

Cheers!